0-common/schema-ddl/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/Schema.scala

```diff
@@ -59,25 +59,6 @@ case class Schema(
   private[iglu] val allProperties = List(multipleOf, minimum, maximum, maxLength, minLength,
     pattern, format, items, additionalItems, minItems, maxItems, properties,
     additionalProperties, required, patternProperties, `type`, enum, oneOf)
-
-  /**
-   * Concise representation of Schema object
-   */
-  override def toString: String = {
```

Comment:
This was the problem with 2.12?

Reply:
Yes @chuwy , these two functions, toString and isEmpty, defined in body of case class Schema were problems for adding 2.12 support.